scripts/convert_csv_to_json.py

```diff
@@ -10,44 +10,48 @@ def generate_json_files():
     source_root = './source-data'
     output_root = './data'
 
+    for source_region_directory_name in os.listdir(source_root):
+        source_region_directory = os.path.join(source_root, source_region_directory_name)
 
-    for source_directory_name in os.listdir(source_root):
+        for language_directory_name in os.listdir(source_region_directory):
+            source_directory = os.path.join(source_region_directory, language_directory_name)
 
-        source_directory = os.path.join(source_root, source_directory_name)
+            for source_file_name in os.listdir(source_directory):
 
-        for source_file_name in os.listdir(source_directory):
+                source_file_location = os.path.join(source_directory, source_file_name)
 
-            source_file_location = os.path.join(source_directory, source_file_name)
+                if os.path.isfile(source_file_location):
 
-            if os.path.isfile(source_file_location):
+                    source_file = open(source_file_location, 'r')
 
-                source_file = open(source_file_location, 'r')
+                    output_file_name = source_file_name.replace('.csv', '.json')
+                    output_file_location = os.path.join(output_root,
+                                                        source_region_directory_name,
+                                                        language_directory_name,
+                                                        output_file_name)
 
-                output_file_name = source_file_name.replace('.csv', '.json')
-                output_file_location = os.path.join(output_root, source_directory_name, output_file_name)
+                    json_data = []
 
-                json_data = []
+                    with open(source_file_location, newline='') as csv_file:
+                        for row in csv.DictReader(csv_file, fieldnames=['term']):
+                            json_data.append(row.get('term'))
+                    try:
+                        with open(output_file_location, 'w', encoding='utf8') as f:
+                            json.dump(json_data, f, indent=3, ensure_ascii=False)
```

Comment:
We should really indent this with the prettier standard amount as per the rest of the repos. I believe it's 2 spaces. 

We should also validate the files created are valid json? There don't appear to be any tests run automatically on each commit?